standalone_scripts/ttalps_plotter_config.py

```diff
@@ -21,43 +59,128 @@
+  "QCD_Pt_3200toInf": 0.0001346,
+  
+  "tta_mAlp-0p35GeV": 1,
+  "SingleMuon2018": 1,
+}
+
 lines = {
   #name                   color
-  "tta_mAlp-0p35GeV"  :  [color_palette[0], ROOT.kSolid],
-  "TTbar_inclusive"   :  [33, ROOT.kSolid],
-  "TTJets"            :  [36, ROOT.kSolid],
+  # "tta_mAlp-0p35GeV"  :  [color_palette[0], ROOT.kSolid],
+  # "TTbar_inclusive"   :  [33, ROOT.kSolid],
+  # "TTJets"            :  [36, ROOT.kSolid],
+  
+  "TTbar_inclusive": [33, ROOT.kSolid],
+  "ST_tW_top": [38, ROOT.kSolid],
+  "ST_tW_antitop": [30, ROOT.kSolid],
+  "ttWJets": [ROOT.kMagenta, ROOT.kSolid],
+  "ttZJets": [41, ROOT.kSolid],
+  "ttHToMuMu": [50, ROOT.kSolid],
+  "QCD_Pt_30to50": [60, ROOT.kSolid],
+  "QCD_Pt_50to80": [61, ROOT.kSolid],
+  "QCD_Pt_80to120": [62, ROOT.kSolid],
+  "QCD_Pt_120to170": [63, ROOT.kSolid],
+  "QCD_Pt_170to300": [64, ROOT.kSolid],
+  "QCD_Pt_300to470": [65, ROOT.kSolid],
+  "QCD_Pt_470to600": [66, ROOT.kSolid],
+  "QCD_Pt_600to800": [67, ROOT.kSolid],
+  "QCD_Pt_800to1000": [68, ROOT.kSolid],
+  "QCD_Pt_1000to1400": [69, ROOT.kSolid],
+  "QCD_Pt_1400to1800": [70, ROOT.kSolid],
+  "QCD_Pt_1800to2400": [71, ROOT.kSolid],
+  "QCD_Pt_2400to3200": [72, ROOT.kSolid],
+  "QCD_Pt_3200toInf": [73, ROOT.kSolid],
+  "tta_mAlp-0p35GeV": [color_palette[0], ROOT.kSolid],
+  "SingleMuon2018": [ROOT.kBlack, ROOT.kSolid],
 } 
 
 legends = {
   #name                   color
-  "tta_mAlp-0p35GeV"  :  "m_{a} = 0.35 GeV",
-  "TTbar_inclusive"   :  "t#bar{t} incl.",
-  "TTJets"            :  "t#bar{t} + jets",
+  # "tta_mAlp-0p35GeV"  :  "m_{a} = 0.35 GeV",
+  # "TTbar_inclusive"   :  "t#bar{t} incl.",
+  # "TTJets"            :  "t#bar{t} + jets",
+  
+  "TTbar_inclusive": "TTbar_inclusive",
+  "ST_tW_top": "ST_tW_top",
+  "ST_tW_antitop": "ST_tW_antitop",
+  "ttWJets": "ttWJets",
+  "ttZJets": "ttZJets",
+  "ttHToMuMu": "ttHToMuMu",
+  "QCD_Pt_30to50": "QCD_Pt_30to50",
+  "QCD_Pt_50to80": "QCD_Pt_50to80",
+  "QCD_Pt_80to120": "QCD_Pt_80to120",
+  "QCD_Pt_120to170": "QCD_Pt_120to170",
+  "QCD_Pt_170to300": "QCD_Pt_170to300",
+  "QCD_Pt_300to470": "QCD_Pt_300to470",
+  "QCD_Pt_470to600": "QCD_Pt_470to600",
+  "QCD_Pt_600to800": "QCD_Pt_600to800",
+  "QCD_Pt_800to1000": "QCD_Pt_800to1000",
+  "QCD_Pt_1000to1400": "QCD_Pt_1000to1400",
+  "QCD_Pt_1400to1800": "QCD_Pt_1400to1800",
+  "QCD_Pt_1800to2400": "QCD_Pt_1800to2400",
+  "QCD_Pt_2400to3200": "QCD_Pt_2400to3200",
+  "QCD_Pt_3200toInf": "QCD_Pt_3200toInf",
+  "tta_mAlp-0p35GeV":"tta_mAlp-0p35GeV",
+  "SingleMuon2018": "SingleMuon2018",
 }
 
-signal_legend_position = (0.7,0.7,0.85,0.75)
-background_legend_position = (0.7,0.75,0.85,0.85)
+legend_types = {
+  "signal": "l",
+  "background": "f",
+  "data": "pl",
+}
+
+legend_position = {
+  "signal": (0.15,0.85,0.25,0.89),
+  "background": (0.7,0.5,0.85,0.85),
+  "data": (0.15,0.8,0.25,0.85),
+}
 
 variables = {
 # key                    title         logy     norm1   rebin    xmin     xmax     ymin,  ymax,  xlabel                 ylabel
```

Comment:
Looking at your plots they are not normalized to 1 right? So the way I intended the bool norm1 it should be false in your case

Reply:
Ok, for the time being, I introduced a string instead of a bool to know which normalization to apply. I also worked a bit on making the normalization procedure a bit clearer (but I'm not 100% happy with it yet, in the future I think I'd like to have a normalizer class that will handle different cases).